src/ZODB/tests/testUtils.py

```diff
@@ -156,6 +156,6 @@ class ExampleClass(object):
 def test_suite():
     suite = unittest.defaultTestLoader.loadTestsFromName(__name__)
     suite.addTest(
-        doctest.DocFileSuite('../utils.txt', checker=checker)
+        doctest.DocFileSuite('../../../doc/utils.rst', checker=checker)
```

Comment:
I'm somewhat concerned that the file movements will cause problems for other projects that extend ZODB's tests. They'll potentially need access to these files if just ZODB is installed, which I don't think they'll get if they're not part of the source tree.

Reply:
I can let them where they were and just include them in the documentation. Would that be better?

Reply:
(I know RelStorage has a tight relationship to the ZODB test suite. I haven't checked to see if moving the files would cause it any problems.)

Reply:
> I can let them where they were and just include them in the documentation. Would that be better?

I suspect that would be safer, yes.

Reply:
Done.